packages/web3-eth-ens/src/lib/ResolverMethodHandler.js

```diff
@@ -22,8 +22,22 @@
 
 var PromiEvent = require('web3-core-promievent');
 var namehash = require('eth-ens-namehash');
+var errors = require('web3-core-helpers').errors;
 var _ = require('underscore');
 
+// These ids obtained at ensdomains docs:
+// https://docs.ens.domains/contract-developer-guide/writing-a-resolver
+var interfaceIds = {
```

Comment:
This should get out-sourced to the config file I think..

Reply:
Fixed in 
a6e19cb